enigma-swing/src/main/java/org/quiltmc/enigma/gui/config/StatsSection.java

```diff
@@ -10,4 +10,12 @@ public class StatsSection extends ReflectiveConfig.Section {
 	public final TrackedValue<String> lastSelectedDir = this.value("");
 	public final TrackedValue<String> lastTopLevelPackage = this.value("");
 	public final TrackedValue<Boolean> shouldIncludeSyntheticParameters = this.value(false);
+	public final TrackedValue<Boolean> shouldCountFallbackNames = this.value(false);
+	public final IconsSection icons = new IconsSection();
+
+	@SerializedNameConvention(NamingSchemes.SNAKE_CASE)
+	public static class IconsSection extends ReflectiveConfig.Section {
```

Comment:
unsure of the best way to handle this. i'm thinking of maybe making this the main config and then having the dialog be transient and open with this as the defaults